getruby.cmd

```diff
@@ -16,10 +16,10 @@ REM Get Ruby and Rails
 REM 32bit
 REM curl -o rubyrails193.zip http://dl.bintray.com/oneclick/rubyinstaller/ruby-2.1.5-i386-mingw32.7z?direct
 REM 64bit
-curl -o ruby215.zip http://dl.bintray.com/oneclick/rubyinstaller/ruby-2.1.5-x64-mingw32.7z?direct
+curl -o ruby215.zip -L http://dl.bintray.com/oneclick/rubyinstaller/ruby-2.1.5-x64-mingw32.7z?direct
 REM Azure puts 7zip here!
 EcHO START Unzipping Ruby
-d:\7zip\7za x -xr!*.ri -y ruby215.zip > rubyout
+SetLocal DisableDelayedExpansion & d:\7zip\7za x -xr!*.ri -y ruby215.zip > rubyout
```

Comment:
I did a bit of Googling and found [this](http://sourceforge.net/p/sevenzip/discussion/45797/thread/37742a46) thread. Specifically this comment:

```
There is some program (or system setting) in your system that doesn't allow "!" in command line.
Try to find that proghram and disable it.
```

It looks like my website and my local machine are treating the ```!``` in a way that causes it to change before being passed to 7zip. Adding ```SetLocal DisableDelayedExpansion & ``` to the start of the line fixes this and now the script works perfectly on my local machine.

I've never heard of ```DisableDelayedExpansion``` before, but [this](http://stackoverflow.com/questions/22278456/enable-and-disable-delayed-expansion-what-does-it-do) SO post seems to answer it to a basic degree.